### ISSUE TITLE
Possibility to have a wait option

### ISSUE THREAD

**Issue by benjamin-taheri** [RobertoFN/streamlit-datalist#3]
I wanted to ask whether it's possible to add an option to the control to wait until exit (or until pressing enter). I need to store the user inputs in the db for future references, and don't want to persist the intermediate values.

As an example, if I want to enter "animals" to the datalist, all "an", "ani", "animal", "animals" get persisted in the db. Right now I added `time.sleep(2)`  after the datalist to address this issue, but it's not a clean solution:

```
previous_queries = fetch_from_db()
st.session_state.text_query = stDatalist(
    '',
    options=previous_queries
)
time.sleep(2)

# The rest of the logic

if st.session_state.text_query != '':
    db_thread = Thread(
        target=save_search_query,
        args=(st.session_state.text_query)
    )
    db_thread.start()
```

I was wondering if we could have an additional parameter in the control that indicates waiting until exit to send over the inputs:
```
previous_queries = fetch_from_db()
st.session_state.text_query = stDatalist(
    '',
    options=previous_queries,
    wait_until_exit=True
)

# The rest of the logic

if st.session_state.text_query != '':
    db_thread = Thread(
        target=save_search_query,
        args=(st.session_state.text_query)
    )
    db_thread.start()
```

**Comment by RobertoFN** [comment]
Hi, this issue is addressed in PR #5. Will update then feature is released in pypi component.

**Comment by RobertoFN** [comment]
Feature released on [v0.0.5](https://pypi.org/project/streamlit-datalist/0.0.5/)
